app/lib/mammoth/channels.rb

```diff
@@ -75,7 +79,7 @@ def list(include_accounts: false)
     # GET channel by id and return all details
     def find(id)
       cache_key = "channels:list:#{id}"
-      Rails.cache.fetch(cache_key, expires_in: 60.seconds) do
+      Rails.cache.fetch(cache_key, expires_in: 1.hour) do
```

Comment:
Just confirming this change from 1 min to 1 hour is expected.

Reply:
same here. channel details should be good from hour to hour